mne/utils/docs.py

```diff
@@ -1861,7 +1861,7 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
         additional argument(s) to your function definition.
 """
 docdict["fun_applyfun"] = applyfun_fun_base.format(
-    " if ``channel_wise=True`` and ``(len(picks), n_times)`` otherwise"
+    " if ``channel_wise=True`` then a 3D array of shape ``(n_epochs, len(picks), n_times)`` is passed, otherwise a 3D array of shape ``(n_epochs, n_channels, n_times)`` is passed"
 )
```

Comment:
This isn't quite right, because this change will be reflected in the documentation for `raw.apply_function` and `evoked.apply_function`, but we don't want it to, since those data will never be 3D!

e.g. see https://output.circle-artifacts.com/output/job/118b9184-1aa0-461c-a3a6-7f831deca1ee/artifacts/0/html/generated/mne.io.Raw.html#mne.io.Raw.apply_function